### PR TITLE
Fixes #1807: [UI 2.0] SearchHelp

### LINKED ISSUE #1807: [UI 2.0] Search Page - SearchHelper
**What would you like to be added**:
For reference, please check #1804 
File can be found here: `src/frontend/next/src/components/SearchHelp.tsx`

### PR DESCRIPTION
**Pull request by rjayroso**
## Issue This PR Addresses
Implements and closes #1807 

## Type of Change

- [ ] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [x] **UI**: Change which improves UI

## Description
As per the new 2.0 UI Design, I have re-styled the search helper. See pictures below:
| Web |
| :---: |
| Capture2 |
[Web Design Reference](https://user-images.githubusercontent.com/60857409/108666981-6acac980-74a6-11eb-8ab1-a721f0e246f6.png)

| Mobile |
| :---: |
| Capture1 |
[ Mobile Design Reference](https://user-images.githubusercontent.com/60857409/108667182-a1a0df80-74a6-11eb-9788-419a0baf6185.png)

See #1804 for complete design reference. Currently, the SearchHelper component is static - I think that it should be handled within the SearchBar/SearchInput components (when the user types the help disappears). Let me know if this isn't the case cc @PedroFonsecaDEV 

## Checklist

- [x] **Quality**: This PR builds and passes our npm test and works locally
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [ ] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

**Comment by izhuravlev** [comment]
@rjayroso I launched the vercel deployment, and after making a request saw this: 
image
I don't think this should work like this

File: src/web/src/components/SearchHelp.tsx [hunk 1]
```diff
@@ -1,72 +1,82 @@
...
-const useStyles = makeStyles(() => ({
-  infoButton: {
-    marginTop: '7.5px',
-    fontSize: '3rem',
-    color: '#8BC2EB',
+const useStyles = makeStyles((theme: Theme) => ({
+  container: {
+    width: '50%',
+    transform: 'translate(-50%, -50%)',
+    position: 'absolute',
```
Comment (izhuravlev) [inline src/web/src/components/SearchHelp.tsx:8]: I am not a CSS God, but I really don't like position: absolute. 
Is there any reasoning why using it here? 
This comment is not a request, more of a comment - I'd personally do it via Flexbox if possible.

**Review by izhuravlev — CHANGES_REQUESTED** [review]
Looks good except for the display error I specified above - try looking into web/src/pages/search.tsx to solve it

File: src/web/src/components/SearchHelp.tsx [hunk 2]
```diff
@@ -1,72 +1,82 @@
...
-}));
-
-const HtmlTooltip = withStyles((theme) => ({
-  tooltip: {
-    backgroundColor: '#f5f5f9',
-    color: 'rgba(0, 0, 0, 0.87)',
-    maxWidth: 500,
-    fontSize: theme.typography.pxToRem(22.5),
-    border: '1px solid #dadde9',
+  subtitle: {
```
[the remark this answers is not in the document]
Comment (rjayroso) [reply src/web/src/components/SearchHelp.tsx:16]: Good catch! It's not needed anymore I'll take it out after I figure out how to work out that display error you found above

[the remark this answers is not in the document]
Comment (izhuravlev) [reply src/web/src/components/SearchHelp.tsx:16]: Sounds good, ping me if you actually need help figuring it out

File: src/web/src/components/SearchHelp.tsx (hunk 1 above)
Comment (huynguyez) [reply src/web/src/components/SearchHelp.tsx:8]: just another way to align the container itself to the centre of its parent, it's okay

Comment (izhuravlev) [reply src/web/src/components/SearchHelp.tsx:8]: okay, this can be resolved than

**Review by humphd — DISMISSED** [review]
This isn't using the right font (Times New Roman), needs to be removed when the loading spinner starts, and when search content is shown.  I'm also not a huge fan of centered text like this.

**Comment by PedroFonsecaDEV** [comment]
@rjayroso Rebase

**Comment by rjayroso** [comment]
> @rjayroso Rebase
> 
> @izhuravlev What do you think about it?
> 
> @huynguyez I think your new design will land after this PR. So I would appreciate your opinion here.
> 
> I think this PR will be a temporary solution to the component.

Sounds good to me! In the meantime, I tried my best to address the issues above.
1. SearchHelp now disappears on submit
2. Font changed to TimesNewRoman
3. Now aligned to the left and uses standard CSS attributes

*Excuse the old search bar below, I haven't pulled the current design yet*

[demo](https://user-images.githubusercontent.com/48869737/111307932-7423fd80-8630-11eb-9041-2fc88be832b2.mp4)

mobile

**Comment by izhuravlev** [comment]
@rjayroso I love it, looks good to me. The only thing that I thought of is that we can probably surround it with a separate "Box". What I mean is that the SearchHelp can be displayed in a rectangle of a different colour, so it is visually separated - but this smth we should ask @PedroFonsecaDEV about.

**Comment by izhuravlev** [comment]
@rjayroso Okay, I just opened the deployment and this is what I saw: 
image

After searching it removes the bottom SearchHelper, but it leaves the top one be. 
Also, not sure if it is related, on small resolutions like iPhone X if no results have been found only the picture is displayed - I can't scroll to the "no Results Found" Message.

**Review by PedroFonsecaDEV — DISMISSED** [review]
Rebase so we can review it with the latest version of our master.

**Comment by izhuravlev** [comment]
So far this looks pretty good. If we don't want to play with the logic a bit and make this being displayed only when the search page is shown for the first time - LGTM. other than that - I would play a bit with the logic here, so the SearchHelp is displayed when there are no results found as well.

**Comment by rjayroso** [comment]
> So far this looks pretty good. If we don't want to play with the logic a bit and make this being displayed only when the search page is shown for the first time - LGTM. other than that - I would play a bit with the logic here, so the SearchHelp is displayed when there are no results found as well.

I think we can merge this and I'll file an issue to improve the SearchHelp logic

**Review by izhuravlev — APPROVED** [review]
LGTM, Don't forget to file a new issue.

**Review by HyperTHD — DISMISSED** [review]
Rebase this, and you can merge this in @rjayroso

**Review by chrispinkney — APPROVED** [review]

**Comment by rjayroso** [comment]
Rebased

**Pull request merged**